qcodes/instrument_drivers/tektronix/AWG5014.py

```diff
@@ -310,29 +337,6 @@ def __init__(self, name, setup_folder, address, reset=False,
                     vals=vals.Numbers(-2.7, 2.7),
                     get_parser=float)
 
-        # # Add functions
```

Comment:
I was wondering what happened to this, is the add functions a deprecated qcodes concept?